tracing/otel/model/converter/converter.go

```diff
@@ -0,0 +1,127 @@
+package converter
+
+import (
+	"strconv"
+
+	"github.com/kiali/kiali/log"
+	jaegerModels "github.com/kiali/kiali/tracing/jaeger/model/json"
+	otel "github.com/kiali/kiali/tracing/otel/model"
+	otelModels "github.com/kiali/kiali/tracing/otel/model/json"
+)
+
+// convertID
+func ConvertId(id string) jaegerModels.TraceID {
+	return jaegerModels.TraceID(id)
+}
+
+// convertID
+func convertSpanId(id string) jaegerModels.SpanID {
+	return jaegerModels.SpanID(id)
+}
+
+// ConvertSpans
+// https://opentelemetry.io/docs/specs/otel/trace/sdk_exporters/jaeger
+func ConvertSpans(spans []otelModels.Span, serviceName string) []jaegerModels.Span {
+	var toRet []jaegerModels.Span
+	for _, span := range spans {
+
+		startTime, err := strconv.ParseUint(span.StartTimeUnixNano, 10, 64)
+		if err != nil {
+			log.Errorf("Error converting start time. Skipping trace")
+			continue
+		}
+
+		duration, err := getDuration(span.EndTimeUnixNano, span.StartTimeUnixNano)
+		if err != nil {
+			log.Errorf("Error converting start time. Skipping trace")
```

Comment:
This error message is identical to the one on line 30. I usually ask that people generate unique error messages everywhere (otherwise you won't know where in the code the error came from - did the error come from this line 36 or line 30?)... even if you just add a period at the end of one but not the other.

However, in this case, I think this is just a copy-n-paste mistake. I think you meant for this to be:
```suggestion
			log.Errorf("Error converting duration. Skipping trace")
```

Either way, this error message should be changed.